src/Microsoft.PowerShell.Commands.Utility/commands/utility/EnableDisableRunspaceDebugCommand.cs

```diff
@@ -362,58 +378,63 @@ protected override void ProcessRecord()
-                            );
+                    WriteError(
+                        new ErrorRecord(new PSInvalidOperationException(string.Format(CultureInfo.InvariantCulture, Debugger.RunspaceOptionInvalidRunspaceState, runspace.Name)),
+                        "SetRunspaceDebugOptionCommandInvalidRunspaceState",
+                        ErrorCategory.InvalidOperation,
+                        this)
+                        );
 
-                        continue;
-                    }
+                    continue;
+                }
 
-                    System.Management.Automation.Debugger debugger = GetDebuggerFromRunspace(runspace);
-                    if (debugger == null)
-                    {
-                        continue;
-                    }
+                System.Management.Automation.Debugger debugger = GetDebuggerFromRunspace(runspace);
+                if (debugger == null)
+                {
+                    continue;
+                }
 
-                    // Enable debugging by preserving debug stop events.
-                    debugger.UnhandledBreakpointMode = UnhandledBreakpointProcessingMode.Wait;
+                // Enable debugging by preserving debug stop events.
+                debugger.UnhandledBreakpointMode = UnhandledBreakpointProcessingMode.Wait;
 
-                    if (this.MyInvocation.BoundParameters.ContainsKey(nameof(BreakAll)))
+                if (this.MyInvocation.BoundParameters.ContainsKey(nameof(BreakAll)))
+                {
+                    if (BreakAll)
                     {
-                        if (BreakAll)
+                        try
                         {
-                            try
-                            {
-                                debugger.SetDebuggerStepMode(true);
-                            }
-                            catch (PSInvalidOperationException e)
-                            {
-                                WriteError(
-                                    new ErrorRecord(
-                                    e,
-                                    "SetRunspaceDebugOptionCommandCannotEnableDebuggerStepping",
-                                    ErrorCategory.InvalidOperation,
-                                    this)
-                                    );
-                            }
+                            debugger.SetDebuggerStepMode(true);
                         }
-                        else
+                        catch (PSInvalidOperationException e)
                         {
-                            debugger.SetDebuggerStepMode(false);
+                            WriteError(
+                                new ErrorRecord(
+                                e,
+                                "SetRunspaceDebugOptionCommandCannotEnableDebuggerStepping",
+                                ErrorCategory.InvalidOperation,
+                                this)
+                                );
```

Comment:
The same. Please move to end of previous line.

Reply:
fixed